src/sidebar/components/Settings/Integrations/Groq/index.tsx

```
@@ -0,0 +1,178 @@
+
+  const { integration, connected, verifyingConnection, verifyConnection } = useGroq()
+
+  const [input, setInput] = React.useState('')
+  const [showApiKey, setShowApiKey] = React.useState(false)
+
+  useEffect(() => {
+    setInput(integration?.apiKey || '')
+  }, [integration])
+
+  const onSave = async () => {
+    if (integration && integration.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)